.github/workflows/no-response.yml

```diff
@@ -0,0 +1,30 @@
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
```

Comment:
You may want to change this one as the latest changes that have been done on `buildozer` ?

Reply:
Done